apps/web/server.js

```diff
@@ -0,0 +1,23 @@
+const { createServer } = require("https");
+const { parse } = require("url");
+const next = require("next");
+const fs = require("fs");
+const port = 3000;
+const dev = process.env.NODE_ENV !== "production";
+const app = next({ dev });
+const handle = app.getRequestHandler();
+
+const httpsOptions = {
+  key: fs.readFileSync("./app.cal.local-key.pem"),
+  cert: fs.readFileSync("./app.cal.local.pem"),
+};
+
+app.prepare().then(() => {
+  createServer(httpsOptions, async (req, res) => {
+    const parsedUrl = parse(req.url, true);
+    await handle(req, res, parsedUrl);
+  }).listen(port, (err) => {
+    if (err) throw err;
+    console.log(`ready - started server on url: https://app.cal.local:${port}`);
```

Comment:
should this be hardcoded like this? are some people not using other local domains?

Reply:
This script is only for development and we have mentioned app.cal.local everywhere in readme

Reply:
<img width="432" alt="Screenshot 2023-11-27 at 10 39 09 PM" src="https://github.com/calcom/cal.com/assets/53316345/eb02d0d3-47cf-4672-882b-a08b8ca2233d">

The issue is that Browsers do not allow camera/mic access on any non-HTTPS hosts except for localhost. (Which i confirmed from daily.co team).

so we need script this to run https://app.cal.local:3000/

Reply:
Ok I have removed the file and added more instructions in readme.